src/components/ChallengesComponent/ChallengeCard/index.js

```diff
@@ -96,14 +96,20 @@ const getPhaseInfo = (c) => {
  * @param onUpdateLaunch
  * @returns {*}
  */
-const hoverComponents = (challenge, onUpdateLaunch) => {
+const hoverComponents = (challenge, onUpdateLaunch, deleteModalLaunch) => {
   const communityAppUrl = `${COMMUNITY_APP_URL}/challenges/${challenge.id}`
   const directUrl = `${DIRECT_PROJECT_URL}/contest/detail?projectId=${challenge.legacyId}`
   const orUrl = `${ONLINE_REVIEW_URL}/review/actions/ViewProjectDetails?pid=${challenge.legacyId}`
 
   // NEW projects never have Legacy challenge created, so don't show links and "Activate" button for them at all
   if (challenge.status.toUpperCase() === CHALLENGE_STATUS.NEW) {
-    return null
+    if (challenge.status.toUpperCase() === CHALLENGE_STATUS.NEW) {
+      return (
+        <button className={styles.deleteButton} onClick={() => deleteModalLaunch()}>
```

Comment:
@mark-nakachon could we replace `() => deleteModalLaunch()` with `deleteModalLaunch`? As the arrow method looks unnecessary here.